src/config/urls.py

```diff
@@ -15,10 +15,30 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.contrib import admin
+from django.urls import path, include
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Atlas-Backend",
+        default_version='v1',
+        description="Atlas-Backend REST API",
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path("state/", include("state.urls")),
     path("district/", include("district.urls")),
     path("data/", include("data.urls")),
+    path('', schema_view.with_ui('swagger', cache_timeout=0),
```

Comment:
**suggestion:** Please add a comment explaining the bottom lines

```suggestion
    
    path('', schema_view.with_ui('swagger', cache_timeout=0),
```

Reply:
We added the comment and removed the /redoc route, as it, like swagger, also generates the api documents, so having two document generators would be redundant